main.cpp

```diff
@@ -52,106 +52,106 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: mana
+//  action 1: check amount of mana
+mana.checkAmount();
+//  action 2: use mana
+mana.useMana();
+//  action 3: refill mana
+mana.refillMana();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: account
+//  action 1: check balance
+account.checkBalance();
+//  action 2: debit funds
+account.debitFunds();
+//  action 3: credit funds
+account.creditFunds();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: distance
+//  action 1: total distance
+distance.totalDistanceTravelled();
+//  action 2: distance remaining
+distance.distanceLeftToGo();
+//  action 3: convert distance to km
+distance.convertToKm();

```

Comment:
This noun doesn't perform these actions.
Something else (a different noun) performs these actions TO or WITH this noun.
- [ ] revise